modules/apps/asset/asset-taglib/src/main/java/com/liferay/asset/taglib/servlet/taglib/AssetTagsSelectorTag.java

```diff
@@ -170,33 +171,29 @@ protected String getEventName() {
 	}
 
 	protected long[] getGroupIds() {
-		if (_groupIds != null) {
```

Comment:
We discussed this earlier on Slack, but I'm still left with the question of why are you removing this? Doesn't this break the logic if we do so?

Reply:
Hi, @wanderlast. I needed to remove it because:

- getGroupIds is just called once when the page is loaded, so this won't cause unnecessary requests.
- We need to get the others group ids related to the provided group id when call the taglib. Only using the given group id itself isn't enough to perform the correct behavior, once it needs the current, ancestor and depot group ids. This also answers your first comment.
- The body of the method is the same used in Categories. [For reference](https://github.com/liferay/liferay-portal/blob/4c3862d7b511114e8a6ad8768602f5c1cdb9ab89/modules/apps/asset/asset-taglib/src/main/java/com/liferay/asset/taglib/servlet/taglib/AssetCategoriesSelectorTag.java#L269).